packages/analytics-v1.1/src/utils/beaconQueue.js

```diff
@@ -58,7 +58,7 @@ class BeaconQueue {
     queue = queue.slice(-(this.maxItems - 1));
     queue.push(message);
     let batch = queue.slice(0);
-    const data = { batch };
+    const data = { batch, sentAt: new Date().toISOString() };
```

Comment:
you ca use teh same import { getCurrentTimeFormatted } from '@rudderstack/analytics-js-common/utilities/timestamp'; here too or is there any blocker?

Reply:
Addressed